UKTrains.indigoPlugin/Contents/Server Plugin/text2png_modern.py

```
@@ -0,0 +1,796 @@
+"""
+Modern mobile-optimized departure board renderer for UK-Trains plugin
+
+This module generates portrait-orientation PNG images optimized for mobile
+messaging apps (WhatsApp, iMessage) using a contemporary card-based design.
+
+Design Specifications:
+    - Dimensions: 414×variable portrait (iPhone compatible)
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)